src/modules/Elsa.Workflows.Sink/Extensions/FeatureExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Elsa.Workflows.Sink.Features;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class FeatureExtensions
+{
+    public static WorkflowSinkFeature UseInMemoryDatabase(this WorkflowSinkFeature feature, Action<MemoryWorkflowSinkPersistenceFeature>? configure = default)
+    {
+        feature.Module.Configure(configure);
+        return feature;
+    }
```

Comment:
I might be missing the point here, but I don't think we need an in-memory sink. The user either installs one or more sinks, or they don't.